Examples/UIExplorer/XHRExample.android.js

```diff
@@ -259,6 +259,85 @@ class FormUploader extends React.Component {
   }
 }
 
+class Headers extends React.Component {
```

Comment:
Can you pull this out into a shared file like XHRExampleHeaders.js so it's not duplicated across platforms?